README.md

```diff
@@ -23,6 +23,17 @@ structures.
 
 Check out our [short-term roadmap](#short-term-roadmap) below for where we're heading!
 
+## Documentation
+
+- Deployment
```

Comment:
Maybe add a sentence:

Check out docs.tigerbeetle.com. Here are a few key pages you might be interested in.

?